src/main/java/cms/gov/madie/measure/services/FhirServicesClient.java

```diff
@@ -61,11 +63,18 @@ public byte[] getMeasureBundleExport(Measure measure, String accessToken) {
   }
 
   public ResponseEntity<HapiOperationOutcome> validateBundle(
-      String testCaseJson, String accessToken) {
+      String testCaseJson, ModelType modelType, String accessToken) {
+    if (modelType == null) {
+      throw new UnsupportedTypeException("Please provide model type.");
+    }
+
+    String modelVersion = modelType.getVersionNumber().replace(".", "-");
     URI uri =
         URI.create(
             fhirServicesConfig.getMadieFhirServiceBaseUrl()
-                + fhirServicesConfig.getMadieFhirServiceValidateBundleUri());
+                + fhirServicesConfig
+                    .getMadieFhirServiceValidateBundleUri()
+                    .replace("$model", modelVersion));
```

Comment:
I believe we could use `UriComponentsBuilder` to build a url from a template but not a blocker

Reply:
thats good idea, creating another PR.

Reply:
here's the PR- https://github.com/MeasureAuthoringTool/measure-service/pull/744